helm/cert-manager-app/values.yaml

```diff
@@ -21,12 +21,14 @@ cainjector:
   image:
 
     # cainjector.image.name
+    # IMPORTANT: this should not be changed.
     name: giantswarm/cert-manager-cainjector
```

Comment:
I was thinking about this before when I first working with Kong... I thought we should hard code this in the actual template rather than making ti a value. I decided against that in Kong as it was easier to test different versions.

In this case, maybe it's better to hard code?

Reply:
yeah maybe a good move - i'll open another PR rather than polluting this one